tests/derive_arc/fails/receiver_box.stderr

```diff
@@ -0,0 +1,5 @@
+error: cannot derive `Rc` for a trait declaring methods with arbitrary receiver types
```

Comment:
This should read *cannot derive `Arc` for a trait...* here as well.

Reply:
This one is strange, as the tests pass...

Reply:
Oh, I see it's the message from the derive above.